### PR TITLE
Added contributing guidelines in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Richard-Strittmatter**
Since our other open source projects have a contributing guide, I thought this one should have one aswell.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+* Create an issue on GitHub,
+if you have found a bug or have an idea for a feature
+* Write test cases for open bug issues
+* Write patches for open bug/feature issues
+
+There are a few guidelines that we need contributors to follow, so that we have a
+chance of keeping on top of things.
+
+* The code must follow the [CakePHP coding standard](http://book.cakephp.org/2.0/en/contributing/cakephp-coding-conventions.html).
```
Comment (ravage84) [inline CONTRIBUTING.md:13]: This schould point to the CakePHP 3.x documentation page

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  * Clearly describe the issue including steps to reproduce, if it is a bug.
+  * Make sure you mention the version and configuration of all software that could be relevant.
+
+## Coding Standard
+
+Make sure your code changes comply with the CakePHP Coding standard,
+using [PHP Codesniffer](https://github.com/squizlabs/PHP_CodeSniffer).
+Execute the following command from the plugin folder:
+
+    vendor/bin/phpcs -p --extensions=php --standard=CakePHP
```
Comment (ravage84) [inline CONTRIBUTING.md:29]: We should promote the ``composer cs-check`` command here.

**Review by ravage84 — CHANGES_REQUESTED** [review]

**Review by ravage84 — APPROVED** [review]
@TerryKern please also review this PR

**Review by TerryKern — APPROVED** [review]
Looks good to me.

**Pull request merged**